GOVERNANCE.md

```diff
@@ -630,10 +630,10 @@ write all the documentation for the PyMC library.
 For current members of the documentation team, refer to the recurrent and
 core contributor membership sections.
 
-### Discourse team
-The focus of the Discourse team is managing and energizing the PyMC Discourse.
+### Community team
+The focus of the Community team is activities intended to nuture, energize, and grow the community of PyMC users and contributors.
 
-For current members of the discourse team, refer to the recurrent and
+For current members of the community team, refer to the recurrent and
 core contributor membership sections.
 
 ### "No-team" tasks
```

Comment:
"no-team" tasks need to be updated to avoid listing community team things.

Reply:
I wasn't sure about the allocation of the no-team tasks.  Given the discussion of the team's mission, I guess these makes sense?

No-team tasks: fundraising and issue triaging
Community: running PyMC related events like PyMCon or sprints, outreach, or presence on social networks.

But then it seemed like the majority of the no-team tasks were being shifted to the community team and we had discussed how things like PyMCon would almost certainly involve many people/teams (social media activity certainly will be).

Reply:
Sounds good. I think it's important to clear expectations on both what teams are supposed and _not supposed_ to do. I also added a clarification on the docs team description for example; its existence doesn't mean only members of the docs team will write documentation or that only members of the docs team will update notebooks to v4. The docs team goal is to maintain the documentation tooling and infrastructure and see if it can be improved, review the docs to see if there are missing parts... but then its everyone that uses said tooling and documents the library. 

I think something similar should happen with pymcon/sprints, and it did already with the sprint. The community team should probably take care of organizing and coordinating the event, like Meenal, Reshama, Beryl and I did for the sprint, but then everyone who can should help with reviewing talk proposals, giving talks, and other tasks "prepared" by the organization team. More or less the same way Ricardo and Austin gave webinars for the sprint and Sayam, Ricardo and Thomas are volunteering for the sprint itself. I wouldn't consider those part of "organizing" even if they are also critical and necessary for the event to suceed.

Does that make sense? 

side note, if you think it would be good to still keep a longer list of "no-team" example tasks I have some ideas: enforcing the governance and code of conduct are also very clear "no team" tasks, also funding allocation and roadmap planning, fundraising can be divided between grant applications and reaching out to potential sponsors too.

Reply:
I have added more info about both the community team activities/responsibilities and tried to limit the intended scope (like you did for the doc team).  I added additional no-team activities as you suggested.

Reply:
Agreed.